base/strings/string.jl

```diff
@@ -30,6 +30,22 @@ Copy a string from the address of a C-style (NUL-terminated) string encoded as U
 
 This function is labelled "unsafe" because it will crash if `p` is not
 a valid memory address to data of the requested length.
+
+# Examples
+```jldoctest
+julia> s = "abcdefghij"
+"abcdefghij"
+
+julia> sp = pointer(s)
+Ptr{UInt8} @0x00007f26aaefe6b8
+
+julia> unsafe_string(sp)
+"abcdefghij"
+
+julia> unsafe_string(sp,5)
```

Comment:
This is only valid in global scope and is not valid on it's own in a function. This function is generally only needed for C interop so a real example involving a real C string would be less misleading. Alternatively, if you make `s` a `Symbol` this code will be at least valid.

Reply:
Could you suggest a simple example of C interop where `unsafe_string` would be needed?

Reply:
There are many in base https://github.com/JuliaLang/julia/search?utf8=%E2%9C%93&q=unsafe_string&type=

E.g. https://github.com/JuliaLang/julia/blob/e009ac62726bfaf0fc43850c985e8f2826496a6b/base/options.jl

Reply:
@yuyichao @nalimilan Thank you very much for your comment and suggest.
I'll try modified them on base your suggest.